opshin/rewrite/rewrite_import.py

```diff
@@ -88,6 +88,8 @@ def visit_ImportFrom(
         if self.filename:
             sys.path.pop()
         module_file = pathlib.Path(module.__file__)
+        if module_file.suffix == ".pyc":
+            module_file = module_file.with_suffix(".py")
```

Comment:
This change was necessary since PyInstaller only bundles `.pyc` versions of opshin files, e.g. `opshin/prelude.py`. This resulted in import errors when trying to compile some contracts using the bundled executable. Please confirm that allowing `.pyc` suffix here can't have any unintended consequences.

Reply:
The restriction is only to make sure we are importing full files - does the compilation actually work with pyc files? Because the parser needs the entire AST of imported modules :thinking:

Reply:
Okay I tested in an environment where the opshin files should not be found automatically and appearently building works - this means that the required "py" file is found correctly. Not sure how but thats for PyInstaller to handle :smile:

Reply:
Yes, not correct what I said about PyInstaller only bundling `.pyc` versions actually. It seems to have both but the import resolves to the `.pyc` version for some reason which (without the introduced changes) make the assert below fail...